.env.defaults

```diff
@@ -5,3 +5,8 @@ JWTAUDIENCE=api.dev
 AWS_ACCESS_KEY_ID=XXXXXXXXXXXXXXXXXXXX
 AWS_SECRET_ACCESS_KEY=xxxxxxxxxxxxxxxxxxxx
 AWS_BUCKET=aws-bucket
+STORAGE_AMAZON_BUCKET=bucket-name
```

Comment:
can we rename this to `HARBOR_REGISTRY_STORAGE_AMAZON_BUCKET` it would explain a bit more what the env variable is used for.

Reply:
Updated in the latest commit